releasenotes/notes/delete-deprecated-varforms-and-featmaps-0ab7bf477eeaa281.yaml

```diff
@@ -0,0 +1,18 @@
+---
+deprecations:
```

Comment:
Should this be a removed section - so that deprecations just list what is newly deprecated

Reply:
Yep -- should be!